keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -436,6 +467,9 @@ _Figure: Topology Manager fetches affinity from hint providers._
 * Guarantee aligned resources for multiple containers in a pod.
 * Refactor to easily support different merge strategies for different policies.
 
+## Beta (v1.19)
+* Support pod-level resource alignment.
```

Comment:
I would put a new line before this to remain consistent with the rest of these sections